examples/epg/rest.tf

```diff
@@ -57,3 +56,12 @@ resource "aci_rest" "rest_taboo_con" {
     "name" = "testcon"
   }
 }
+
+resource "aci_rest" "rest_abs_graph" {
+  path       = "api/node/mo/${aci_tenant.tenant_for_contract.id}/AbsGraph-testgraph.json"
+  class_name = "vnsAbsGraph"
+
+  content = {
+    "name" = "testgraph"
+  }
+}
```

Comment:
This is now available in a dedicated resource see: aci_l4_l7_service_graph_template